gpt_researcher/skills/image_generator.py

```
@@ -0,0 +1,767 @@
+                llm_provider=self.cfg.fast_llm_provider,
+                stream=False,
+                websocket=None,
+                max_tokens=1500,
+                llm_kwargs=self.cfg.llm_kwargs,
+            )
+            
+            # Parse the response
+            image_suggestions = self._parse_analysis_response(response, sections)
+            return image_suggestions[:self.max_images]
+            
+        except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)